content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -122,6 +122,171 @@ see the [API reference](/docs/reference/kubernetes-api/) for more information. I
 is not possible to access sub-resources across multiple resources - generally a new
 virtual resource type would be used if that becomes necessary.
 
+## HTTP media types {#alternate-representations-of-resources}
```

Comment:
Might not matter for this PR, but @benluddy is working on support for CBOR as an additional content type.

There are a few implications.  The most obvious is that there will be another content type to document.  The other is that clients may *send* CBOR to an apiserver that does not support it and that apiserver will return a HTTP 415 (I believe, correct me if I'm wrong @benluddy).

Reply:
Great! Let's merge this so there's an obvious place to add the CBOR details.

IMO, merging this would make CBOR easier to document.